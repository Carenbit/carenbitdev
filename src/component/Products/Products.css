.placeHolderImage {
  width: 85%;
  height: 85%;
  object-fit: cover;
  border-radius: 10px;
  box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.1);
}

.productCard {
  margin: 5%;
  width: 40%;
  min-height: 70%;
  border-radius: 10px;
  box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.1);
  background-color: #fff9f0;
  gap: 10px;
}

.productCard:hover {
  box-shadow: 0 0 10px 0 rgba(0, 0, 0, 0.3);
  transition: 0.5s;
  scale: 1.1;
}

.pHeading {
  font-size: 40px;
  font-weight: 600;
  line-height: 54px;
  letter-spacing: -0.05em;
}

.pNames {
  font-size: 24px;
  font-weight: 600;
  line-height: 29px;
  letter-spacing: -0.05em;
}

@media screen and (max-width: 464px) {
  .productCard {
    width: 80%;
    margin: 5% 10%;
  }
  .pHeading {
    font-size: 30px;
    line-height: 40.5px;
  }
  .pNames {
    font-size: 20px;
  }
}
