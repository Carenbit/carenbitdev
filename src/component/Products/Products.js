import "./Products.css";
import placeHolderImage from "../../assets/placeholder.png";

const Products = () => {
  return (
    <div>
      <h1 className="text-center pHeading">Our Products</h1>
      <div className="d-flex flex-wrap pContainer">
        <div className="d-flex flex-column p-3 productCard align-items-center">
          <img src={placeHolderImage} alt="..." className="placeHolderImage" />
          <h4 className="pNames">Product name</h4>
          <p className="fw-light text-center fs-6">
            Lörem ipsum susanne Jonasson Anette Nilsson medan Axel Ahmed Adam
            Jönsson. Charlotta Ek Alice Arvidsson i David{" "}
          </p>
        </div>
        <div className="d-flex flex-column p-3 productCard align-items-center">
          <img src={placeHolderImage} alt="..." className="placeHolderImage" />
          <h4 className="pNames">Product name</h4>
          <p className="fw-light text-center fs-6">
            Lörem ipsum susanne Jonasson Anette Nilsson medan Axel Ahmed Adam
            Jönsson. Charlotta Ek Alice Arvidsson i David{" "}
          </p>
        </div>
        <div className="d-flex flex-column p-3 productCard align-items-center">
          <img src={placeHolderImage} alt="..." className="placeHolderImage" />
          <h4 className="pNames">Product name</h4>
          <p className="fw-light text-center fs-6">
            Lörem ipsum susanne Jonasson Anette Nilsson medan Axel Ahmed Adam
            Jönsson. Charlotta Ek Alice Arvidsson i David{" "}
          </p>
        </div>
        <div className="d-flex flex-column p-3 productCard align-items-center">
          <img src={placeHolderImage} alt="..." className="placeHolderImage " />
          <h4 className="pNames">Product name</h4>
          <p className="fw-light text-center fs-6">
            Lörem ipsum susanne Jonasson Anette Nilsson medan Axel Ahmed Adam
            Jönsson. Charlotta Ek Alice Arvidsson i David{" "}
          </p>
        </div>
      </div>
    </div>
  );
};

export default Products;
