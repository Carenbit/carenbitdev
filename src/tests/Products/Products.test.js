import "@testing-library/jest-dom";
import { render, screen } from "@testing-library/react";
import Products from "../../component/Products/Products.js";

test("should have a heading with text 'Our Products'", () => {
  render(<Products />);
  const heading = screen.getByText(/Our Products/i);
  expect(heading).toBeInTheDocument();
});

test("should match screenshot of Products component", () => {
  const { container } = render(<Products />);
  expect(container).toMatchSnapshot();
});
