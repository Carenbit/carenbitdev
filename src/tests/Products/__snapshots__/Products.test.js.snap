// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match screenshot of Products component 1`] = `
<div>
  <div
    id="productsPage"
  >
    <h1
      class="text-center pHeading"
    >
      Our Products
    </h1>
    <div
      class="d-flex flex-wrap pContainer"
    >
      <div
        class="d-flex flex-column p-3 productCard align-items-center"
      >
        <img
          alt="..."
          class="placeHolderImage"
          src="ProductImg1.png"
        />
        <h4
          class="pNames"
        >
          Streamline
        </h4>
        <p
          class="fw-light text-center fs-6"
        >
          Lörem ipsum susanne Jonasson Anette Nilsson medan Axel Ahmed Adam Jönsson. Charlotta Ek Alice Arvidsson i David
           
        </p>
      </div>
      <div
        class="d-flex flex-column p-3 productCard align-items-center"
      >
        <img
          alt="..."
          class="placeHolderImage"
          src="ProductImg2.png"
        />
        <h4
          class="pNames"
        >
          Automate Workflows
        </h4>
        <p
          class="fw-light text-center fs-6"
        >
          Lörem ipsum susanne Jonasson Anette Nilsson medan Axel Ahmed Adam Jönsson. Charlotta Ek Alice Arvidsson i David
           
        </p>
      </div>
      <div
        class="d-flex flex-column p-3 productCard align-items-center"
      >
        <img
          alt="..."
          class="placeHolderImage"
          src="ProductImg3.png"
        />
        <h4
          class="pNames"
        >
          Product name
        </h4>
        <p
          class="fw-light text-center fs-6"
        >
          Lörem ipsum susanne Jonasson Anette Nilsson medan Axel Ahmed Adam Jönsson. Charlotta Ek Alice Arvidsson i David
           
        </p>
      </div>
      <div
        class="d-flex flex-column p-3 productCard align-items-center"
      >
        <img
          alt="..."
          class="placeHolderImage "
          src="ProductImg4.png"
        />
        <h4
          class="pNames"
        >
          Product name
        </h4>
        <p
          class="fw-light text-center fs-6"
        >
          Lörem ipsum susanne Jonasson Anette Nilsson medan Axel Ahmed Adam Jönsson. Charlotta Ek Alice Arvidsson i David
           
        </p>
      </div>
    </div>
  </div>
</div>
`;
